pulumi/grapl/__main__.py

```diff
@@ -306,8 +306,7 @@ def main() -> None:
             # The vars with a leading underscore indicate that the hcl local version of the variable should be used
             # instead of the var version.
             _redis_endpoint=cache.endpoint,
-            container_registry="docker.cloudsmith.io/",
-            container_repo="raw/",
+            container_repository=f"{config.container_repository()}/",
```

Comment:
Is defining a pulumi.config.get as a function in infra config our preferred pattern?

Reply:
🤔 I can probably dispense with the function... I think it actually did some more stuff in an earlier iteration, but not anymore.

Reply:
On second thought, I'll leave it as is, since we call it in a couple places.